src/core/renderer/hostDescriptors/descriptors/geometries/geometry.ts

```diff
@@ -44,6 +49,16 @@ class GeometryDescriptor extends WrappedEntityDescriptor<GeometryWrapper,
       instance.vertices = newValue;
       instance.verticesNeedUpdate = true;
     }, false, true);
+
+    this.hasProp<Face3[]>("faces", (instance, newValue, oldProps, newProps) => {
+      if (instance.faces.length !== newValue.length) {
+        this.remountTrigger(instance, newValue, oldProps, newProps);
```

Comment:
I saw that `vertices` required a `remountTrigger` so I did it here as well. When does a prop require a `remountTrigger`?

Reply:
Properties for some types in `three` are cached in such a way that if you would like to modify them, you need to replace it completely. Sometimes it's also easier to just recreate the whole thing instead of going through a complicated path to replace all parts properly. A simple way to check is if nothing happens when you modify the property. If so, you may need to remount. Can investigate further to see how a property is used by the core parts of `three`.

In this case, if a geometry's number of vertices change, this may help explain it specifically: https://stackoverflow.com/questions/14840026/dynamically-adding-vertices-to-a-line-in-three-js